nav2_smac_planner/lattice_primitives/tests/trajectory_visualizer.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) 2021, Matthew Booker 
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License. Reserved.
+
+import json
+from pathlib import Path
```

Comment:
Bump:  I think this tests file needs a bit more explanation as to its role / purpose / need / use. Just add a block comment on top about what this is doing and where its used